src/components/Choropleth.vue

```diff
@@ -1,28 +1,32 @@
 <template>
-    <LMap ref="map" style="height: 100vh; width: 100%" @ready="updateBounds">
-        <LTileLayer data-testid="tile-layer" v-bind="backgroundLayer"></LTileLayer>
-        <LGeoJson
-            v-for="f in featuresWithColours"
-            ref="featureRefs"
-            :key="getFeatureId(f.feature)"
-            :data-testid="getFeatureId(f.feature)"
-            :geojson="f.feature"
-            :options="createTooltips"
-            :options-style="
-                () => {
-                    return { ...style, fillColor: f.colour };
-                }
-            "
-        >
-        </LGeoJson>
-    </LMap>
+    <div>
+        <LMap ref="map" style="height: 100vh; width: 100%" @ready="updateBounds">
+            <LTileLayer data-testid="tile-layer" v-bind="backgroundLayer"></LTileLayer>
+            <LGeoJson
+                v-for="f in featuresWithColours"
+                ref="featureRefs"
+                :key="getFeatureId(f.feature)"
+                :data-testid="getFeatureId(f.feature)"
+                :geojson="f.feature"
+                :options="createTooltips"
+                :options-style="
+                    () => {
+                        return { ...style, fillColor: f.colour };
+                    }
+                "
+            >
+            </LGeoJson>
+        </LMap>
+        <div style="visibility: hidden" class="choropleth-data-summary" v-bind="dataSummary"></div>
```

Comment:
I think you added this just for testing, right? So we don't want screen-readers to try to interpret it? In which case it needs 'display: none' or just the HTML 'hidden' attribute

https://a11y-guidelines.orange.com/en/articles/accessible-hiding/

Reply:
Wait I'm wrong, visibility: hidden works fine for screen readers (omits the element for all types of users)